extras/shared-drafts.rst

```diff
@@ -0,0 +1,138 @@
+﻿Shared Drafts
+=============
+
+Share drafts for new or existing tickets with other agents of your group.
+Load, modify and safe an existing draft e.g. as QA process with colleagues.
+
+.. figure:: /images/extras/shared-drafts/ticket-zoom-with-activated-shared-draft-function.png
+   :alt: Screenshot showing the sub menu of the Update button with macros and an
+         option to save the draft
+   :align: center
+
+   By using ᐱ next to the "Update" button Zammad will provide you with an option
+   to save your draft (and thus share it) if the option is enabled.
+
+.. note:: **🤔 Huh? I don’t see “Share Draft” option...** 
+
+   This feature is **optional**; if you don’t see it in the menu,
+   that means your administrator disabled this function.
+   Administrators can learn more in our `admin documentation`_.
+
+.. _admin documentation:
+   https://admin-docs.zammad.org/en/latest/manage/groups/settings.html
+
+.. hint:: **🤓 Let's be clear up front**
+
+   Zammad technically has two draft functions:
+
+      * shared draft (allow others to load the draft), and
+      * user drafts.
+
+   | Every time you change a tickets field, Zammad will safe this in your
+     personal draft. You then can share this as a shared draft if you want to.
+   | User drafts are *always* available!
+
+Handling drafts
+---------------
+
+.. note::
+
+   Shared draft handling is the same for existing tickets and new tickets. 🙏
+
+Load a draft
+   .. figure:: /images/extras/shared-drafts/loading-a-shared-draft-ticket-zoom.gif
+      :alt: Screencast showing how to open the shared draft load dialogue within
+            the ticket zoom
+
+   Loading a draft works the same from the point of draft previews.
```

Comment:
This sentence makes no sense to me. Can you please clarify?

Reply:
This sentence takes direct context to the load draft screencast above.
The screencast shows only how to load a draft within ticket zoom because I wanted to reduce documentation size.

If that's not good enough and confusing, I'll redo the screencast to include both?

Reply:
I don't understand what "point of draft previews" means.

Reply:
The draft preview is thsi screen:

![image](https://user-images.githubusercontent.com/6549061/158145539-67db8e57-73f6-47c0-857d-1299d60514a6.png)


From this point on, the load draft functionality behaves exactly the same in both, new tickets and existing ones.
That's what I wanted to say.

Reply:
Suggestion:
Loading a draft works the same starting from draft previews.